resources/scripts.js

```diff
@@ -2,88 +2,184 @@ function firstGame() {
     if (firstViewed === true) {
         firstScreenFade.show();
     }
+    return;
 }
 
 function newGame() {
     firstViewed = false;
+    lastPlayerMoney = playerMoney;
     playerMoney = 100;
-    displayMoney.innerHTML = playerMoney;
+    moneyChange(0);
+    resetGame();
+    displayMessage(NEWGAME);
+    return;
+}
+
+function resetGame() {
+    playerRoll = 1;
+    pointOpen = false;
+    buttonPosition(playerRoll);
     betAmount = 0;
-    buttonPosition(1);
-    displayMessage('New game started');
-    rollButtonState(false);
-    betButtonState(true);
+    totalBets = 0;
+    buttonStates();
+    return;
 }
 
 function saveGame() {
-    displayMessage('Saving not yet implemented');
+    displayMessage(SAVEGAME);
     offButton.style.visibility = 'hidden';
     onButton.style.left = '315px'; // Table border: 15px, square: 230px, mid: 85px
+    return;
 }
 
 function loadGame() {
-    displayMessage('Loading not yet implemented');
+    displayMessage(LOADGAME);
     offButton.style.visibility = 'visible';
     onButton.style.left = '85px';
+    return;
 }
 
-function rollButtonState(enabled) {
-    if (enabled === true) {
-        rollElement.disabled = false;
+function buttonStates() {
+    if (betAmount === 0) {
+        rollButtonElement.disabled = true;
+        betDialogElement.disabled = false;
+        betButtonElement.disabled = false;
+        comeButtonElement.disabled = true;
         return;
     }
-    rollElement.disabled = true;
-}
-
-function betButtonState(enabled) {
-    maxBetInput.setAttribute('max', playerMoney);
-    displayMoney.innerHTML = playerMoney;
-    if (enabled === true) {
-        betDialog.disabled = false;
-        displayBetButton.disabled = false;
-        return;
+    betDialogElement.setAttribute('max', playerMoney);
+    lastPlayerMoney = playerMoney;
+    moneyChange(0);
+    rollButtonElement.disabled = false;
+    betDialogElement.disabled = true;
+    betButtonElement.disabled = true;
+    comeButtonElement.disabled = true;
+    if (pointOpen === true) {
+        comeButtonElement.disabled = false;
+        betDialogElement.disabled = false;
     }
-    betDialog.disabled = true;
-    displayBetButton.disabled = true;
+    return;
 }
 
-function firstRoll() {
+async function firstRoll() {
     diceRoll();
-    displayMessage(`You rolled ${playerRoll}`);
+    await new Promise(resolve => setTimeout(resolve, 1000));
+    buildSummary(DICEROLLED);
     if (playerRoll === 7 || playerRoll === 11) {
-        playerWin();
+        payOut(true, true, betAmount);
         return;
     }
     if (playerRoll === 2 || playerRoll === 3 || playerRoll === 12) {
-        playerLose();
+        payOut(true, false);
         return;
     }
     playerPoint = playerRoll;
     pointOpen = true;
     buttonPosition(playerRoll);
-    displayMessage(`Your point is ${playerPoint}`);
```

Comment:
Message to let player know that Place bets can now be made by clicking on the board should be added. HTML next to "Amount" would be a good place that is noticeable.

Reply:
Added notifications.